internal/device/darwin.go

```diff
@@ -44,3 +65,31 @@ func Notify() {
 		log.Fatal(err)
 	}
 }
+
+// utils
+
+func isCharged(status string) bool {
+	return strings.EqualFold(status, "charged")
```

Comment:
This is the sort of thing I mean about string matching being vulnerable. You can just imagine, can't you, that Apple decides to change the output format to say something like:

```
Your battery is nowhere near charged
```

and now your library suddenly starts giving incorrect results, even though you haven't changed the code at all.

Reply:
resolved in 1ccc772349311a8c9c50441562461c48885f8418